xlsx/src/export/worksheets.rs

```diff
@@ -90,8 +148,11 @@ pub(crate) fn get_worksheet_xml(
         row_style_dict.insert(row.r, row.clone());
     }
 
+    const ROW_STRING_COUNT_BEFORE_WRITE: usize = 10;
```

Comment:
Empirically determined batch size for calling `write!` on rows. So we'll accumulate 10 rows worth of data before writing it. Smaller and I saw longer run times, larger and we use more memory with the same runtime.